net/devif/devif_poll.c

```diff
@@ -849,29 +798,7 @@ int devif_poll(FAR struct net_driver_s *dev, devif_poll_callback_t callback)
 int devif_timer(FAR struct net_driver_s *dev, int delay,
                 devif_poll_callback_t callback)
 {
-#if defined(NET_TCP_HAVE_STACK)
-  int hsec = TICK2HSEC(delay);
-#endif
-  int bstop = false;
-
-#ifdef NET_TCP_HAVE_STACK
-  /* Traverse all of the active TCP connections and perform the
-   * timer action.
-   */
-
-  bstop = devif_poll_tcp_timer(dev, callback, hsec);
-#endif
-
-  /* If possible, continue with a normal poll checking for pending
-   * network driver actions.
-   */
-
-  if (!bstop)
-    {
-      bstop = devif_poll(dev, callback);
-    }
-
-  return bstop;
+  return devif_poll(dev, callback);
```

Comment:
so `delay` parameter now is ignored?

Reply:
> so `delay` parameter now is ignored?

yes, we will remove all devif_timer in the next change.